### PR TITLE
import-beats: adjust generated files

### PR DESCRIPTION
**Pull request by mtojek**
Changes in `import-beats`:
* fields: add default type "keyword" when missing
* change input title from "Collecting..." to "Collect"
* regenerate files

### CODE REVIEW

**Comment by mtojek** [comment]
@ruflin Could you please take a look at this one? ;)

File: dev/import-beats/fields.go [hunk 1]
```diff
@@ -141,9 +141,22 @@ func loadFieldsFile(path string) ([]fieldDefinition, error) {
...
 	}
+	fs = loadDefaultFieldValues(fs)
 	return fs, nil
 }
 
+func loadDefaultFieldValues(fs fieldDefinitionArray) fieldDefinitionArray {
+	var withDefaults fieldDefinitionArray
+	for _, f := range fs {
+		if f.Type == "" {
+			f.Type = "keyword"
```
Comment (ruflin) [inline dev/import-beats/fields.go:152]: +1 on this. Less magic in the fields.yml and will make it easier to also validate the fields.yml files.

File: dev/packages/beats/aws-0.0.1/kibana/map/dae24080-739a-11ea-a345-f985c61fe654.json [hunk 2]
```diff
@@ -28,7 +28,7 @@
...
...obalQuery\":true,\"indexPatternRefName\":\"layer_1_source_index_pattern\"},\"style\":{\"type\":\"VECTOR\",\"properties\":{\"icon\":{\"type\":\"STATIC\",\"options\":{\"value\":\"marker\"}},\"fillColor\":{\"type\":\"STATIC\",\"options\":{\"color\":\"#54B399\"}},\"lineColor\":{\"type\":\"STATIC\",\"options\":{\"color\":\"#41937c\"}},\"lineWidth\":{\"type\":\"STATIC\",\"options\":{\"size\":1}},\"iconSize\":{\"type\":\"STATIC\",\"options\":{\"size\":6}},\"iconOrientation\":{\"type\":\"STATIC\",\"options\":{\"orientation\":0}},\"labelText\":{\"type\":\"STATIC\",\"options\":{\"value\":\"\"}},\"labelColor\":{\"type\":\"STATIC\",\"options\":{\"color\":\"#000000\"}},\"labelSize\":{\"type\":\"STATIC\",\"options\":{\"size\":14}},\"labelBorderColor\":{\"type\":\"STATIC\",\"options\":{\"color\":\"#FFFFFF\"}},\"symbolizeAs\":{\"options\":{\"value\":\"circle\"}},\"labelBorderSize\":{\"options\":{\"size\":\"SMALL\"}}},\"isTimeAware\":true},\"id\":\"a10fa758-30ad-4e2a-bf9d-472e133a7f17\",\"label\":\"CloudTrail Soure Location\",\"minZoom\":0,\"maxZoom\":24,\"alpha\":0.75,\"visible\":true,\"type\":\"VECTOR\",\"joins\":[],\"query\":{\"query\":\"event.dataset:aws.cloudtrail\",\"language\":\"kuery\"}}]",
```
Comment (ruflin) [inline dev/packages/beats/aws-0.0.1/kibana/map/dae24080-739a-11ea-a345-f985c61fe654.json:31]: Unrelated to this PR but stumbled over this. I have no idea what layerListJSON is but I'm wondering if it is also something we should add to our encode / decode list for better versioning. Same for mapStateJSON.

Comment (mtojek) [reply dev/packages/beats/aws-0.0.1/kibana/map/dae24080-739a-11ea-a345-f985c61fe654.json:31]: Nice spotting, will look into this in the next PR. Thanks!

**Pull request merged**